config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -0,0 +1,27 @@
+---
+ca:
+  gobierto_dashboards:
+    available_cards: Targetes disponibles
+    changes: Dashboard modificat
+    date: 'Data: '
+    default_title: Títol del dashboard
+    delete: Esborrar
+    instructions: |
+      "⇧ Selecciona un tipus de targeta i inclou-la en l'àrea central ➞
+
+      Després podràs configurar-la."
+    objective: Objectiu
+    percentage: Assoliment
```

Comment:
Progres?

Reply:
_Consecució_, perhaps, to keep consistency with spanish

Reply:
ok!

Reply:
Done